jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/DefaultCredentialRetrieversTest.java

```diff
@@ -39,82 +39,88 @@
 @RunWith(MockitoJUnitRunner.class)
 public class DefaultCredentialRetrieversTest {
 
-  @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
+  @Rule public final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-  @Mock private CredentialRetrieverFactory mockCredentialRetrieverFactory;
-  @Mock private CredentialRetriever mockDockerCredentialHelperCredentialRetriever;
-  @Mock private CredentialRetriever mockKnownCredentialRetriever;
-  @Mock private CredentialRetriever mockInferredCredentialRetriever;
-  @Mock private CredentialRetriever mockInferCredentialHelperCredentialRetriever;
-  @Mock private CredentialRetriever mockDockerConfigCredentialRetriever;
+  @Mock private CredentialRetrieverFactory credentialRetrieverFactory;
+  @Mock private CredentialRetriever dockerCredentialHelperCredentialRetriever;
+  @Mock private CredentialRetriever knownCredentialRetriever;
+  @Mock private CredentialRetriever inferredCredentialRetriever;
+  @Mock private CredentialRetriever inferCredentialHelperCredentialRetriever;
+  @Mock private CredentialRetriever dockerConfigCredentialRetriever;
+  @Mock private CredentialRetriever applicationDefaultCredentialRetriever;
```

Comment:
It would be better to leave these type of renames to separate PRs.  There was a preference to explicitly indicate mocked objects with a `mock` prefix, though it's not documented in the `STYLE_GUIDE.md`